caravel/models.py

```diff
@@ -250,6 +252,34 @@ def slice_link(self):
         return '<a href="{url}">{obj.slice_name}</a>'.format(
             url=url, obj=self)
 
+    def get_viz(self, url_params_multidict=None):
+        """Creates BaseViz object from the url_params_multidict.
+
+        Parameters
```

Comment:
If we want to auto generate docs for the code we should stick with Sphinx's (the documentation framework we use) way of specifying docstrings:
http://www.sphinx-doc.org/en/stable/domains.html#info-field-lists